test/abilities/shields-down.test.ts

```diff
@@ -54,4 +56,130 @@ describe("Abilities - SHIELDS DOWN", () => {
 
     expect(minior.formIndex).toBe(meteorForm);
   });
+
+  test("should ignore non-volatile status moves", async () => {
+    game.override.enemyMoveset([MoveId.SPORE]);
+
+    await game.classicMode.startBattle([Species.MINIOR]);
+    game.move.use(MoveId.SPLASH);
+    await game.phaseInterceptor.to(TurnEndPhase);
```

Comment:
```suggestion
    await game.toEndOfTurn();
```
same goes for all other instances of this line